CHANGELOG.md

```diff
@@ -3,6 +3,85 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.16.0
+_Sep 28, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Added `clearable` prop to DatePickers (#9095) @noraleonte
```

Comment:
Maybe you have ideas for improving the phrasing, but maybe we should detail this a bit? 🤔  Also, it's not exactly a prop on the Pickers, but rather a behavior extended from a prop on the field 

cc. @LukasTy @flaviendelangle 

```suggestion
- 🎁 Add `clearable` behavior to the Pickers (#9095) @noraleonte

The Pickers now have an out-of-the box implementation for clearing the field value. You can see the documentation for this behavior on the [Date Picker documentation](https://mui.com/x/react-date-pickers/date-picker/#clearing-the-value).
```

Reply:
Yes, the suggestion totally makes sense.
I've updated the suggestion to have two spaces for the explanation sentence, which will end up grouping it together with the same list item. 👌